scripts/regenerate_protos.sh

```diff
@@ -9,32 +9,20 @@ set -e -o pipefail
 echo "Regenerating Protocol Buffers from Rippled"
 
 # Directory to write generated code to (.js and .d.ts files)
-JS_OUT_DIR="./build/generated"
-TS_OUT_DIR="./generated"
+OUT_DIR="./src/generated"
 
 PROTO_PATH="./rippled/src/ripple/proto/"
 
-mkdir -p $TS_OUT_DIR
-mkdir -p $JS_OUT_DIR
+mkdir -p $OUT_DIR
 
 # Generate node code.
 $PWD/node_modules/grpc-tools/bin/protoc \
-    --js_out=import_style=commonjs,binary:$TS_OUT_DIR \
-    --js_out=import_style=commonjs,binary:$JS_OUT_DIR \
-    --grpc_out=$TS_OUT_DIR \
-    --grpc_out=$JS_OUT_DIR \
+    --js_out=import_style=commonjs,binary:$OUT_DIR \
+    --grpc-web_out=import_style=commonjs+dts,mode=grpcwebtext:$OUT_DIR \
```

Comment:
Does the `dts` here mean we still ship typings?

Reply:
Yes the protos will produce a commonjs js file and a typescript typings file along with it.